app/lib/nomis_client/image.rb

```diff
@@ -5,7 +5,9 @@ module NomisClient
   # :nocov:
   class Image < NomisClient::Base
     class << self
-      def get booking_id
+      def get(person)
+        booking_id = person.latest_profile.latest_nomis_booking_id
```

Comment:
not happy with this change - it ties the Nomis::Image client to our person model (additional coupling) - all our other Nomis::<x> modules are independent, so this is inconsistent as well

Reply:
I got your point, it was an attempt to clean a bit the controller,
since that's a query, I'll move it in the model.

Reply:
This isnt a model - it's a service to get images from NOMIS

Reply:
I've created a services for that.